lisa/target_script.py

```diff
@@ -112,6 +112,7 @@ def run(self, as_root=False, background=False, timeout=None):
             if background:
                 self._bg_shell = self._target.background(self._remote_path,
                                                     as_root=self._run_as_root)
+                return self._bg_shell
```

Comment:
That's quite ugly, maybe we should go for separate `run()` & `run_background()` methods

Reply:
I don't think that's an issue since nobody is relying on the return value currently (as it is None).